3ds Max/Max2Babylon/Forms/AnimationForm.cs

```diff
@@ -45,7 +45,7 @@ private void AnimationForm_Load(object sender, EventArgs e)
         private void createAnimationButton_Click(object sender, EventArgs e)
         {
             AnimationGroup info = new AnimationGroup();
-            
+
```

Comment:
can we please minimize white space changes not related to code changes. it makes git-blame associate you with stuff you may not have changed substantially.

I always appreciate code cleanup, but I'd like to avoid hygienic changes that aren't part of a behavioral change.